arwen/contexts/managedType.go

```diff
@@ -37,6 +37,8 @@ const p256CurveUnmarshalCompressedMultiplier = 100
 const p384CurveUnmarshalCompressedMultiplier = 200
 const p521CurveUnmarshalCompressedMultiplier = 400
 
+const minEncodedBigFloatNumArgs = 6
```

Comment:
It's rather the minimum length of a single encoded big float, in bytes. Consider the following rename:
```suggestion
const minEncodedBigFloatLength = 6
```